### PR TITLE
Announce 4.166.0

### PR DESCRIPTION
**Pull request by Atry**

### CODE REVIEW

File: _posts/2022-08-10-hhvm-4.166.markdown [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+
+HHVM 4.166 is released! 4.163 - 4.165 remain supported, as do the 4.128 and
+4.153 LTS releases.
+
+# Dropping support for Ubuntu 21.04 and 21.10
+
+This release does not compile on Ubuntu 21.04 and 21.10. Both versions
+have reached End of Life earlier this year according to
+[this post](https://fridge.ubuntu.com/2022/01/21/ubuntu-21-04-hirsute-hippo-end-of-life-reached-on-january-20-2022/)
+and [that
```
Comment (AndrewDiMola) [inline _posts/2022-08-10-hhvm-4.166.markdown:16]:
```suggestion
and [this
```

**Review by AndrewDiMola — APPROVED** [review]

**Comment by AndrewDiMola** [comment]
@Wilfred - Does this change the docs re: expression trees in any significant way?

> - Lambdas and calls in Expression Trees are now virtualized:
     - Lambdas are now wrapped in `lambdaType(...)`.
     - Calls are now translated to `__unwrap()` method instead of direct invoking.
     
https://docs.hhvm.com/hack/expression-trees/syntax-supported#lambdas

Related... what does it mean for lambdas and calls to be virtualized? Anything to do with runtime visibility? 
image

**Comment by Atry** [comment]
@AndrewDiMola Yes, I think it is a significant change. See comment messages in https://github.com/facebook/hhvm/commit/e7777169b4a0bb770f2e9e9b1602f1938741af28

**Comment by Atry** [comment]
We are making progress in implicit contexts. I did not mention it in the release note because I suppose it is still an unstable feature.

**Pull request merged**